snap/snapcraft.yaml

```diff
@@ -843,18 +853,21 @@ parts:
   app-service-config:
     plugin: nil
     stage-snaps:
-      - edgex-app-service-configurable/2.0/stable
+      - edgex-app-service-configurable/2.1/stable
     stage:
+      - -res/external-mqtt-trigger/configuration.toml
```

Comment:
Add a comment to mention: `Exclude every profile other than the "rules-engine":` 
Or optionally, rewrite the rule to include what is needed: https://snapcraft.io/docs/snapcraft-filesets

Reply:
Updated. 

FYI, I tried to include files rather than exclude them, but It only works with `filesets` keyword, because include and exclude files have a conflict with each other. However, `filesets` brings some errors such as below and I wasn't been able to solve. 
```
Failed to generate snap metadata: The specified command 
'bin/app-service-configurable -cp -r -s -confdir $SNAP_DATA/config/app-service-configurable/res -profile rules-engine' 
defined in the app 'app-service-configurable' does not exist.
```
```
+ snapcraftctl stage
[Errno 2] No such file or directory: '/root/parts/app-service-config/install/res'
```